app/services/ai-integration-services/integration-client.tsx

```
@@ -50,28 +50,28 @@ export default function IntegrationClient() {
                             <p className="text-lg text-muted-foreground leading-relaxed mb-8">
                                 For organisations with strict data residency, security policies, or regulatory constraints, we design the gateway first: VPC-deployable, auditable, model-agnostic, and owned by you. Switch providers without rewriting the workflow.
                             </p>
-                            <p className="text-lg text-white/80 font-mono">
+                            <p className="text-lg text-foreground-muted font-mono">
                                 No replatforming. No shadow AI. No brittle "glue code."
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)